src/peft/mixed_model.py

```diff
@@ -83,18 +83,10 @@ def _check_config_compatible(peft_config: PeftConfig) -> None:
 
 class PeftMixedModel(PushToHubMixin, torch.nn.Module):
     """
-    Peft model for mixing different types of adapters.
+    PeftMixedModel for loading mixing different types of adapters for inference.
 
-    This class currently does not support saving and loading. Instead, it is assumed that the adapters are already
-    trained and loading the model requires a script to be run each time.
-
-    Currently, the main purpose of mixed adapter types is to combine trained adapters for inference. Although it is
```

Comment:
Why remove the information here and in the docstring below?

Reply:
This information is already in the [mixed model docs](https://huggingface.co/docs/peft/main/en/developer_guides/mixed_models#tips). I added a short link here so users can refer to that guide.